inventory/all_projects/nomad

```diff
@@ -2,32 +2,62 @@
 nomad-host-prod1.lib.princeton.edu
 nomad-host-prod2.lib.princeton.edu
 nomad-host-prod3.lib.princeton.edu
+
 [nomad_clients_production]
 nomad-client-prod1.lib.princeton.edu
 nomad-client-prod2.lib.princeton.edu
 nomad-client-prod3.lib.princeton.edu
 nomad-client-prod4.lib.princeton.edu
+
 [nomad_servers_staging]
 nomad-host-staging1.lib.princeton.edu
 nomad-host-staging2.lib.princeton.edu
 nomad-host-staging3.lib.princeton.edu
+
 [nomad_clients_staging]
 dpul-collections-staging1.lib.princeton.edu
 dpul-collections-staging2.lib.princeton.edu
 nomad-client-staging1.lib.princeton.edu
 nomad-client-staging2.lib.princeton.edu
 nomad-client-staging3.lib.princeton.edu
+
+[nomad_servers_sandbox]
+nomad-host-sandbox1.lib.princeton.edu
+nomad-host-sandbox2.lib.princeton.edu
+nomad-host-sandbox3.lib.princeton.edu
+
+[nomad_clients_sandbox]
+nomad-client-sandbox1.lib.princeton.edu
+nomad-client-sandbox2.lib.princeton.edu
+
 [nomad_clients:children]
 nomad_clients_staging
 nomad_clients_production
+nomad_clients_sandbox
+
 [nomad_servers:children]
 nomad_servers_production
 nomad_servers_staging
+nomad_servers_sandbox
+
 [nomad_production:children]
 nomad_servers_production
 nomad_clients_production
+
+[nomad_staging:children]
+nomad_servers_staging
+nomad_clients_staging
+
+[nomad_sandbox:children]
+nomad_servers_sandbox
+nomad_clients_sandbox
+
+# Keep the original nomad_cluster for backwards compatibility
+# since I am unclear what this will break
 [nomad_cluster:children]
```

Comment:
Should we have a `nomad_cluster_sandbox` and an appropriate group_var for that too? That way there's no overlap.